plugins/modules/nfs_exports_info.py

```diff
@@ -0,0 +1,133 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+# Copyright: (c) 2025, Samaneh Yousefnezhad <s-yousefenzhad@um.ac.ir>
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = r"""
+---
+module: nfs_exports_info
+
+short_description: Extract folders, IPs, and options from /etc/exports
+
+description: >
+  This module retrieves and processes the contents of the /etc/exports file from a remote server,
+  mapping folders to their corresponding IP addresses and access options.
```

Comment:
Using paragraphs makes it easier to extend this:
```suggestion
description:
  - This module retrieves and processes the contents of the C(/etc/exports) file from a remote server,
    mapping folders to their corresponding IP addresses and access options.
```